obs/window-basic-main.cpp

```diff
@@ -80,6 +80,11 @@ OBSBasic::OBSBasic(QWidget *parent)
 {
 	ui->setupUi(this);
 
+	QFile File("stylesheet.qss");
```

Comment:
Variables should be camelCase, e.g.:
```cpp
QFile file(…);
```

Also you probably want to load the stylesheet from the (user) config directory as opposed to the current directory